sky/templates/spot-controller.yaml.j2

```diff
@@ -2,14 +2,10 @@
 
 name: {{cluster_name}}
 
-# The resources will be defined in sky.spot_launch
+# The default resources is defined in sky.spot_utils.DEFAULT_SPOT_CONTROLLER_RESOURCES,
+# and will be assigned to the spot controller task in `sky.spot_launch`.
 # resources:
 #   disk_size: 50
```

Comment:
A little confused.  IIUC, this `disk_size` is for the controller VM, and is it being picked up on the backend now? What if user uncomments this line and changes it to a different number?

Reply:
Ahh, good point. It will be set in the backend now. Let me remove the `resources` field, only leave the comment there.